src/main/java/frc/auto/AutoBuilder.java

```diff
@@ -26,22 +26,22 @@ public static AutoBuilder getInstance() {
     }
 
     private AutoBuilder(){
-       writer = new CSVWriter(Constants.rootDirectory);
+       writer = new CSVWriter(Constants.ROOT_DIRECTORY);
        driveIO = DriveIO.getInstance();
-       driveIO.stopAllOutputs();
     }
 
     public void setStartRecording() {
         startTime = System.currentTimeMillis();
     }
     
     public void recordData() {
-
+        System.out.println("Made it to recordData()!");
```

Comment:
Is this needed for the code, or was that just for debugging, if debugging it should be removed from the code

Reply:
Just debugging, will remove it!